.devcontainer/codespaces/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "nextflow-training (codespaces)",
+
+    // Update this to pin the release tag before release
+    "image": "ghcr.io/nextflow-io/training:latest",
+    // Uncomment this and comment "image" when testing Dockerfile changes
+    // "build": { "dockerfile": "../Dockerfile", "context": ".." },
```

Comment:
Why are we doing this? Because it has to "build" every time?

Couldn't we utilize prebuilds? https://docs.github.com/en/codespaces/prebuilding-your-codespaces/about-github-codespaces-prebuilds

Reply:
Not with a free GitHub org, sadly.